src/libcore/convert.rs

```diff
@@ -113,9 +113,6 @@ pub const fn identity<T>(x: T) -> T { x }
 /// - Use `Borrow` when the goal is related to writing code that is agnostic to
 ///   the type of borrow and whether it is a reference or value
 ///
-/// See [the book][book] for a more detailed comparison.
-///
-/// [book]: ../../book/first-edition/borrow-and-asref.html
 /// [`Borrow`]: ../../std/borrow/trait.Borrow.html
```

Comment:
I cannot find any equivalent link to this comparison in the new book.

Reply:
cc @steveklabnik

Reply:
That’s correct, there’s not, which is why removing it is the right call.